HW04.Rmd

```diff
@@ -1,126 +1,186 @@
+
+#Then, I created a summary table counting the number of deaths per season. The season with the highest numer of deaths due tu gun violence is summer:
+kable(gun_deaths %>% count(season, sort=TRUE),format="markdown",
+      col.names=c("Season","Numer of gun deaths"))
+```
+
+# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
+
+### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
+
+Whites that are killed by guns are more likely to die because of suicide. In contrast, black and hispanics who are killed by guns are more likely to die because of homicide. See tables below, with the percentage of total deaths by intent for each race:
+
+```{r race_2}
+#Below, I created summary tables with the number of deaths by intent for each race. I also calculated the percentage for each intent. I realize that my strategy here is a bit convoluted, so I very much appreacite any suggestions here!
+
+# Whites
+white_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("White")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(white_intent_count,format="markdown")
+
+# Blacks
+black_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("Black")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(black_intent_count,format="markdown")
+
+# Hispanics
+hispanic_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("Hispanic")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(hispanic_intent_count,format="markdown")
+
+```
+
+### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
+
+Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
+
+(I changed the order a little here. I performed statistical test after the corresponding figure instead of doing it at the end)
+
+#### Are there differences in age depending on wether police was involved or not?
+```{r police 1}
+gun_deaths$police <- as.factor(gun_deaths$police)
+
+#Boxplots showing age in cases were police was involved and was not involved
+#Age seems to be a little bit higher for cases in which the police was not involved (0)
+ggplot(gun_deaths,mapping = aes(x = police, y= age)) +
+  geom_boxplot(,fill="gray80")+
+  theme_bw()+
+  theme(axis.title = element_text(color="black", size=12))+
+  xlab("Police involvement?")
+
+#And based on a t-test the difference appears to be statistically significant. Gun death victims are statistically older in cases in which the police was not involved.
```

Comment:
Interesting! I got the opposite: gun death victims are younger when the police were not involved. However, I filtered out suicide as a cause of death, probably that was the difference?